jerry-core/ecma/base/ecma-globals.h

```diff
@@ -1085,19 +1083,90 @@ typedef struct
                                                         *   so the chunk area is enlarged by one for this value */
 } ecma_collection_chunk_t;
 
+/**
+ * Direct string types (2 bit).
+ */
+typedef enum
+{
+  ECMA_DIRECT_STRING_PTR = 0, /**< string is a string pointer, only used by property names */
```

Comment:
In some parts of the code we align the enum comments and in some other parts we do not. What is the correct style?

Reply:
The unaligned is the correct. But some old code has aligned (mostly parser), and I don't want to change it in this patch.